### DISCUSSION TITLE
Vercel is down??

### DISCUSSION THREAD

**Discussion by afern247** [vercel/community#5506] [Help]
### Page to Investigate

https://road2crypto.com/home

### Steps to Reproduce

I'm getting an error on [my website](https://road2crypto.com/home) for a deployment that was working fine 27 days ago and I haven't redeployed, what's going on?

![image](https://github.com/vercel/community/assets/34192856/cdf35dfb-8f5d-44a3-8e5d-d4464cc1706a)

Is vercel down or something?

**Comment by afern247** [comment]
I just redeployed the same version that was working fine since 27 days ago and still the error. Where can I get intimidate support for this as a paid member?

**Comment by charleywolf** [comment]
Also having this issue on all of my sites.

**Comment by afern247** [comment]
New error:

![image](https://github.com/vercel/community/assets/34192856/524b1045-f2c7-4284-a032-85d59a097f4a)

**Comment by swarnava** [comment]
The issue should be fixed now. https://www.vercel-status.com/incidents/3ly94g3wbqtl
